docs/bugzilla.md

```diff
@@ -107,3 +108,16 @@ users.
 ```python
 t.add_cc(['username1@mail.com', 'username2@mail.com'])
 ```
+
+### add_attachment(self, data, file_name, summary, \*\*kwargs ) <a name="add_attachment"></a>
+
+Add attachment in a Bugzilla ticket. Keyword arguments are used to 
+specify ticket fields if any.
```

Comment:
Change to:

Keyword arguments are used to specify additional attachment options.